# GPS trajectory to be read and executed by DJI waypoint_navigator
velocity_range: 5

# Trajectory
# x (ENU) [m] / longitude (GPS) [deg]
easting: [8.680281, 8.681361, 8.681435, 8.680306, 8.680374, 8.681481]
# y (ENU) [m] / latitude (GPS) [deg]
northing: [47.453184, 47.453562, 47.453454, 47.453082, 47.452992, 47.453382]
# z (ENU) [m] / altitude (GPS) [m]
height: [1.2, 1.2, 1.2, 1.2, 1.2, 1.2]
# Heading [deg]
heading: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
# Waiting time at each waypoint [s]
waiting_time: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
